Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2005-02-22
Initial Package Version: 3.3.2
Origin: Jim Gifford
Upstream Status: N/A
Description: Fixes Compile Issue with 1.1.6 speex
 
--- kdemultimedia-3.3.2/akode/plugins/xiph_decoder/speex_decoder.cpp.orig	2005-02-23 04:32:18.041937352 +0000
+++ kdemultimedia-3.3.2/akode/plugins/xiph_decoder/speex_decoder.cpp	2005-02-23 04:32:47.689430248 +0000
@@ -25,10 +25,10 @@
 extern "C" {
 #include <string.h>
 #include <stdlib.h>
-#include <speex.h>
-#include <speex_header.h>
-#include <speex_callbacks.h>
-#include <speex_stereo.h>
+#include <speex/speex.h>
+#include <speex/speex_header.h>
+#include <speex/speex_callbacks.h>
+#include <speex/speex_stereo.h>
 #include <ogg/ogg.h>
 }
 
